shell.nix

```diff
@@ -1,12 +1,12 @@
-(import
-  (
-    let lock = builtins.fromJSON (builtins.readFile ./flake.lock); in
-    fetchTarball {
-      url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
-      sha256 = lock.nodes.flake-compat.locked.narHash;
-    }
-  )
-  { src = ./.; }
+(import (
+  let
+    lock = builtins.fromJSON (builtins.readFile ./flake.lock);
+  in
+  fetchTarball {
+    url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+    sha256 = lock.nodes.flake-compat.locked.narHash;
+  }
+) { src = ./.; }
   # Since a lot of existing CI tests is based on `shell.nix`
   # we forward to the integrationTests shell, instead of the
   # default (developer) shell.
```

Comment:
This shell is broken anyway. It references an integrationTests shell but that doesn't exist.

Reply:
Let's fix in a follow-up PR